src/lib.rs

```diff
@@ -122,21 +122,19 @@ mod tests {
             .add(BoardCommand::Swap([1, 2].into(), [2, 2].into()))
             .unwrap();
 
-        let mut update_stage = SystemStage::parallel();
-        update_stage.add_system(read_commands);
+        let mut update_stage = App::new();
```

Comment:
Since `update_stage` is now no longer a stage but an app, it'd make sense to rename it to `app` in these tests